packages/react-composites/src/composites/common/VideoEffectsPane.tsx

```diff
@@ -150,6 +179,14 @@ const VideoEffectsPaneTrampoline = (
       isLightDismiss={true}
       className={mergeStyles(headerStyles)}
     >
+      {showError && latestEffectError && (
```

Comment:
if we have both a warning and an error - do we want to show both or let the error take precedence?

Reply:
In my opinion, we should show both as user has option to dismiss the error and we do want to show warning in case camera is turned off. Lets check with AlexP what does he suggest:
<img width="600" alt="Screenshot 2023-05-01 at 11 51 38 AM" src="https://user-images.githubusercontent.com/97124699/235512656-08a703d0-ae33-4a0a-9a20-67766b7aca59.png">